### DISCUSSION TITLE
Next Steps - what should we be working on now?

### DISCUSSION THREAD

**Discussion by dilgreen** [CredComSoc/ProtocolWorkingGroup#4] [General]
Three to five things to focus on - mostly immediate blocks to /requirements for progress, but as necessary, work to prepare for future blocks/requirements.

**Comment by dilgreen** [comment]
A while ago, I suggested that the next step should be a zoom call to discuss the 'two track' approach.

I didn't follow through to arrange that, and things stalled a bit.  Apologies.

Looking at time zones (@JohnG-cc  I don't know where you are, exactly - guessed Sydney for timezone), it looks as if [a two+1/2 hour band can work for all of us](https://www.worldtimebuddy.com/?qm=1&lid=2643743,2179537,2147714,3173721&h=2643743&date=2023-7-17&sln=8.5-11&hf=1). Early morning for us in the UK, late evening for Miles in NZ, hopefully more straightforward for Matt and John.

Can I suggest three options? Indicate availability below.

_(I tried the 'make an issue from this discussion', but it A/wasn't necessary - anyone can use click boxes here, even in a comment they didn't author - which is nice), and B/ it only carried the Discussion topic entry across - not comments. I closed the issue. Ignore any notifications.)_

Suggest alternatives if you can't make any.

Thursday Jul 20th, 9am UK, 8pm NZ, 6PM Aus, 10am It
- [x]  Dil
- [ ] Tom
- ~[ ]~ Miles
- [x] John
- [x] Matt

Sunday Jul 30th, 9am UK, 8pm NZ, 6PM Aus, 10am It
- [x]  Dil
- [ ] Tom
- [ ] Miles
- [x] John
- [x] Matt

Tuesday Aug 1st, 9am UK, 8pm NZ, 6PM Aus, 10am It
- [x]  Dil
- [ ] Tom
- [ ] Miles
- [x] John
- [x] Matt

**Reply by matslats** [reply]
all good with me.

**Comment by JohnG-cc** [comment]
any of these is good for me too.

**Comment by utunga** [comment]
These times are tricky as (a) I can't do the firs tone due to prior engagement (b) I will be traveling/at a conference for the second two times. I wonder if ya'll go ahead with the Aug 1st time and I will joing via zoom if i can  - depending on where I am at and what the internet is like...?

**Reply by dilgreen** [reply]
Seeing as:

1. we need you
2. the rest of us seem flexible, 

Is there maybe a day before Aug 1 where you could do an 8pm NZ time or is the pre-conference time just too stressed?

In which case, what's the first day you'll be back (and recovered)?

**Comment by utunga** [comment]
OK so the thing is I will be at the icml.cc conference in Honolulu that week so 9am UK time is 10pm there.

Thinking out loud approx one week later might work for me.. 

- Wed Jul 26th, 9am UK, 10pm Honolulu, 6PM Aus, 10am ?

Seems likely that I'll just be chilling in my hotel at that point. Unfortunately until I get there it's hard to know for sure how things are going to work but I can pencil this time in..  does that work for others?

Feel free to reply in text or with a thumbs up emoji if it works for you.
